htdocs/map_galaxy.php

```diff
@@ -97,6 +97,7 @@
 		}
 	}
 
+	$galaxy->getSectors(); //optimized call to cache all sectors first
```

Comment:
It would make sense imo to put this in `SmrGalaxy->getMapSectors` when `$dist == false || ($dist * 2 + 1 >= $galaxy-> getHeight() && $dist * 2 + 1 >= $galaxy->getWidth())` so that it will automatically be used whenever it makes sense in both local and galaxy map

Reply:
Yep, I completely agree. My plan was to optimize `getMapSectors` itself so that it would benefit from using a single multi-sector query regardless of the function arguments or galaxy size, in which case the calls to `getSectors` aren't needed. I suppose I should just make your suggested change now in case I don't finish my plan as soon as I expect.

Reply:
It's probably worth doing it quickly, you've seen all the bits of code from things I planned to do.. :P